src/ice_shelf/MOM_ice_shelf.F90

```diff
@@ -264,7 +274,10 @@ subroutine shelf_calc_flux(sfc_state_in, fluxes_in, Time, time_step_in, CS)
 
   real, dimension(SZI_(CS%grid),SZJ_(CS%grid)) :: &
     exch_vel_t, &  !< Sub-shelf thermal exchange velocity [Z T-1 ~> m s-1]
-    exch_vel_s     !< Sub-shelf salt exchange velocity [Z T-1 ~> m s-1]
+    exch_vel_s, &  !< Sub-shelf salt exchange velocity [Z T-1 ~> m s-1]
+    tmp, &         !< Temporary field used when calculating diagnostics
```

Comment:
Please add the units of these variables to these descriptions.  For variables that are reused with various different units, the description might be `[various]`.